packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -1125,6 +1139,10 @@ class SemanticsNode extends AbstractNode with DiagnosticableTreeMixin {
   /// If this rect is null [parentSemanticsClipRect] also has to be null.
   Rect parentPaintClipRect;
 
+  /// The index of this node with respect to its parent, including any hidden
+  /// siblings.
```

Comment:
nit: I do not understand this dartdoc :)

What does "with respect to" mean when talking about an index? And how can siblings be included with respect to the parent? :confused: 

Is being hidden determined by some property? Could you link to that property from the doc?